docs/CodeDevelopment/test.md

```diff
@@ -1,18 +1,20 @@
 # Tests
 
-Testing firmware is critical and should be done so much more than it is today. So please, start writing tests. A lot
-of work has been done to make it easier.
+Testing firmware is critical and testing should go beyond what is performed today. With more thorough testing,
+adoption of code updates, code refactoring and maintenance all become easier.  As bugs are reported and fixed,
+new test cases can be added to ensure continued compliance.  So please, start writing tests. A lot of work
+has been done to make adding tests easier.
```

Comment:
Should a link to the UnitTestFrameworkPkg be added here? Or one of the other points you talk about unit tests?

Reply:
links to the UnittestFrameworkPkg are already in test.md under UEFI Unit Tests - C code.

I changed to make them more prominent.